.github/workflows/deploy-standalone-plugins.yml

```diff
@@ -69,6 +69,9 @@ jobs:
       - name: Install npm dependencies
         run: npm ci
 
+      - name: Check plugin versions
+        run: npm run versions -- --plugin=${{ matrix.plugin }}
```

Comment:
@thelovekesh Anything you'd change about this?

Reply:
Looks like a great addition to me.